source/upgrade/notify-admin.rst

```diff
@@ -0,0 +1,25 @@
+Notify admin
+============
+
+In some cases there are Mattermost features that are limited to specific plans. When using Mattermost, users who come across these unavailable features can send a notification to System Admins indicating that they'd like access to this feature.
+
+If access to a feature requires a plan upgrade, System Admins receive notifications of these requests in order to collect data before upgrading. For example, if only one end-user requests access to AD/LDAP, it probably isn't necessary to upgrade. However, if 20 users, including Team and Channel Admins request it, you may want to consider upgrading to support this need.
+
+This feature is designed for informational purposes only, and no action is required unless you want to take action. Your Mattermost instance's functionality is not affected if you choose not to upgrade.
+
+Notifications
+-------------
+
+Notifications are triggered by users. The very first user trigger results in a bot message to the System Admin, indicating the feature or functionality the user has requested. This bot message is listed in the direct message section of the channel sidebar.
```

Comment:
@johndavidlugtu, does this go to all System Admins or just one? If all, what happens if one decides to upgrade and the others don't want to? Is there an upgrade notification sent in that message? If there are multiple admins, who gets the message first?

Reply:
@justinegeffen good questions!

It goes to all System Admins. @AGMETEOR can provide more details for the other questions

Reply:
@justinegeffen 

> If all, what happens if one decides to upgrade and the others don't want to?

One admin upgrading affects the entire workspace. We don't foresee a case where there is a degraded user experience when upgrading.

> Is there an upgrade notification sent in that message?

Once the workspace is upgraded, there is no notification. That said, admins can see in-product multiple places that state their current plan.

> If there are multiple admins, who gets the message first?

They should receive it all simultaneously. Allan can confirm